src/model/response/qtum/QtumBlock.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ *
+ * @export
+ * @interface QtumBlock
+ */
+export interface QtumBlock {
+    result: {
```

Comment:
no result and error. Please move the data like hash to the highest hierarchy.

Reply:
ok